sfe/overridesimporter.go

```diff
@@ -0,0 +1,276 @@
+		if err != nil {
+			return nil, "", fmt.Errorf("getting/validating accountURI: %w", err)
+		}
+		accountID, err := accountURIToID(accountURI)
+		if err != nil {
+			return nil, "", fmt.Errorf("parsing accountURI to accountID: %w", err)
+		}
+		bucketKey, err := rl.BuildBucketKey(rl.NewOrdersPerAccount, accountID, identifier.ACMEIdentifier{}, identifier.ACMEIdentifiers{}, netip.Addr{})
+		if err != nil {
+			return nil, "", fmt.Errorf("building bucket key: %w", err)
+		}
+		req = makeReq(rl.NewOrdersPerAccount, bucketKey, organization, tier)
+		accountDomainOrIP = accountURI
+
+	case rl.CertificatesPerDomainPerAccount.String():
+		accountURI, err := im.getValidatedFieldValue(fields, AccountURIFieldName, "")
+		if err != nil {
+			return nil, "", fmt.Errorf("getting/validating accountURI: %w", err)
+		}
+		accountID, err := accountURIToID(accountURI)
+		if err != nil {
+			return nil, "", fmt.Errorf("parsing accountURI to accountID: %w", err)
+		}
+		bucketKey, err := rl.BuildBucketKey(rl.CertificatesPerDomainPerAccount, accountID, identifier.ACMEIdentifier{}, identifier.ACMEIdentifiers{}, netip.Addr{})
+		if err != nil {
+			return nil, "", fmt.Errorf("building bucket key: %w", err)
+		}
+		req = makeReq(rl.CertificatesPerDomainPerAccount, bucketKey, organization, tier)
+		accountDomainOrIP = accountURI
+
+	case rl.CertificatesPerDomain.String() + perDNSNameSuffix:
+		dnsName, err := im.getValidatedFieldValue(fields, RegisteredDomainFieldName, rateLimit)
+		if err != nil {
+			return nil, "", fmt.Errorf("getting/validating registeredDomain: %w", err)
+		}
+		bucketKey, err := rl.BuildBucketKey(rl.CertificatesPerDomain, 0, identifier.NewDNS(dnsName), identifier.ACMEIdentifiers{}, netip.Addr{})
+		if err != nil {
+			return nil, "", fmt.Errorf("building bucket key: %w", err)
+		}
+		accountDomainOrIP = dnsName
+		req = makeReq(rl.CertificatesPerDomain, bucketKey, organization, tier)
+
+	case rl.CertificatesPerDomain.String() + perIPSuffix:
+		ipAddrStr, err := im.getValidatedFieldValue(fields, IPAddressFieldName, rateLimit)
+		if err != nil {
+			return nil, "", fmt.Errorf("getting/validating ipAddress: %w", err)
+		}
+		ipAddr, err := netip.ParseAddr(ipAddrStr)
+		if err != nil {
+			return nil, "", fmt.Errorf("parsing ipAddress: %w", err)
+		}
+		bucketKey, err := rl.BuildBucketKey(rl.CertificatesPerDomain, 0, identifier.NewIP(ipAddr), identifier.ACMEIdentifiers{}, netip.Addr{})
+		if err != nil {
+			return nil, "", fmt.Errorf("building bucket key: %w", err)
+		}
+		req = makeReq(rl.CertificatesPerDomain, bucketKey, organization, tier)
+		accountDomainOrIP = ipAddrStr
+
+	default:
+		return nil, "", fmt.Errorf("unknown rate limit")
+	}
+	return req, accountDomainOrIP, nil
+}
+
+func (im *OverridesImporter) processTicket(ctx context.Context, ticketID int64, fields map[string]string) error {
+	req, accountDomainOrIP, err := im.makeAddOverrideRequest(fields)
+	if err != nil {
+		// This will recur until the operator corrects the ticket.
```

Comment:
How often will this recur? Are we going to wind up spamming if we get a malformed ticket?

Reply:
Ah! Now I get it. Let's update the comment to say what actually happens, e.g.:


```suggestion
		// Move to "pending" so the next tick won't comment again.
```